getting_started/6_Understanding_Qubits_and_Quantum_Gates.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<blockquote>Two possible basis states for a qubit are the states |0\rangle and |1\rangle</blockquote><p>The MathJax isn't rendering correctly; enclose <code>|0\\rangle</code> and <code>|1\\rangle</code> in <code>$</code>.</p><p> </p><p>Also, for some column vectors don't seem to be rendering properly on GitHub.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
updated